services/backend/src/database/migrations/20191031145350-create-user-and-profile.js

```diff
@@ -152,7 +153,7 @@ module.exports = {
         });
       });
   },
-  down: (queryInterface) => {
```

Comment:
seems to be cause by eslint. please just confirm if this styling chnage is correct (im not sure)